src/components/sbb-card/sbb-card.scss

```diff
@@ -0,0 +1,118 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-3x);
+  --sbb-card-padding-block-end: var(--sbb-card-padding-block-start);
+}
+
+:host([size='s']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='m']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host([size='l']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-4x);
+}
+
+:host([size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-xxs);
+}
+
+:host([size='xxl']) {
+  --sbb-card-padding-inline: var(--sbb-spacing-responsive-s);
+  --sbb-card-padding-block-start: var(--sbb-spacing-responsive-s);
+}
+
+:host(.sbb-card--has-badge[size='m']),
+:host(.sbb-card--has-badge[size='l']),
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host(.sbb-card--has-badge[size='m']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxxs);
+}
+
+:host(.sbb-card--has-badge[size='l']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-fixed-4x);
+}
+
+:host(.sbb-card--has-badge[size='xl']) {
+  --sbb-card-padding-block-end: var(--sbb-spacing-responsive-xxs);
+}
+
+.sbb-card {
+  position: relative;
+  overflow: hidden;
+  border-radius: var(--sbb-border-radius-4x);
+  transition: box-shadow var(--sbb-animation-duration-6x) var(--sbb-animation-easing);
```

Comment:
For the border radius it would make sense to create a css var because it is used in sbb-card, before and button as well.
also for the animation timing and duration a css var would be a nice option :)